.github/workflows/ci.yml

```diff
@@ -223,13 +190,14 @@ jobs:
         # other ecosystems with -v into the updater image we can still run those tests.
         run: |
           docker run --env DEPENDABOT_TEST_ACCESS_TOKEN \
+            --pull never \
             --env VCR \
             --rm \
             -v "$(pwd)/updater/spec/fixtures/vcr_cassettes:/home/dependabot/dependabot-updater/spec/fixtures/vcr_cassettes" \
             -v "$(pwd)/npm_and_yarn:/home/dependabot/npm_and_yarn" \
             -v "$(pwd)/go_modules:/home/dependabot/go_modules" \
             -v "$(pwd)/composer:/home/dependabot/composer" \
-            "dependabot/dependabot-updater-bundler:latest" bundle exec rspec
+            "ghcr.io/dependabot/dependabot-updater-bundler:latest" bundle exec rspec
```

Comment:
This would previously be run through `script/ci-test-updater`, should that be restored too?

Reply:
Restored ✅ 

Also I noticed script/test was the same as script/ci-test-updater, so I deleted it since "ci-test-updater" is more precise with what it does.